app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -11,7 +11,8 @@ def do_clone_task_check(clone_task_ref)
 
   def find_destination_in_vmdb(vm_uid_hash)
     ems_ref = vm_uid_hash.values.join("\\")
-    ManageIQ::Providers::Azure::CloudManager::Vm.find_by(:ems_ref => ems_ref.downcase)
+    rest = ManageIQ::Providers::Azure::CloudManager::Vm.where("lower(ems_ref) = ?", ems_ref.downcase)
+    rest && rest.first
```

Comment:
Use `find_by` and reduce the two lines to:
`ManageIQ::Providers::Azure::CloudManager::Vm.find_by("lower(ems_ref) = ?", ems_ref.downcase)`